tools/build_defs/shell_toolchain/toolchains/access.bzl

```diff
@@ -34,8 +34,13 @@ def call_shell(shell_context, method_, *args):
 
     return result
 
+def _quoted(arg):
+    return arg.startswith("\"") and arg.endswith("\"")
```

Comment:
Technically, `arg.endswith("\"")` could also be true for non-quoted literals, if we also have `arg.endswith("\\\"")`. But in that case the arg could be half-quoted?

Also I'm not sure how this deals with mixed quotation and whatnot.